projects/usbguard/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER allenwebb@google.com
+
+RUN apt-get update && apt-get install -y \
+  make \
+  autoconf \
+  automake \
+  libtool \
+  pkg-config \
+  libxml2-utils \
+  xsltproc \
+  libqb-dev \
+  libprotobuf-dev \
```

Comment:
Why are we downloading the protobuf stuff? I looked at the PR this depends on and it didn't seem like any of them used libprotobuf-mutator. 

Are these packages being installed so that you can use them in the future, or does usbguard depend on them?

Reply:
OK. No need to compile anything out.